lib/handle-pull-request.ts

```diff
@@ -26,12 +27,14 @@ import {
 export default async function handlePullRequest(): Promise<void> {
   if (!process.env.GITHUB_TOKEN) {
     core.setFailed("GITHUB_TOKEN environment variable is not set");
-    process.exit(1);
+    return;
   }
 
   const octokit = github.getOctokit(process.env.GITHUB_TOKEN);
 
-  const eventPayload = github.context.payload as PullRequestEvent;
+  const eventPayload = JSON.parse(
```

Comment:
I'm doing the same logic as the `@actions/github` package does here so we can mock the `process.env.GITHUB_EVENT_PATH` for testing purposes.